client/src/com/mirth/connect/client/ui/actions/PasteAction.java

```diff
@@ -35,8 +36,14 @@ public void actionPerformed(ActionEvent e) {
     public boolean isEnabled() {
         if (comp.isVisible() && comp.isEditable() && comp.isEnabled()) {
             try {
-                Transferable contents = Toolkit.getDefaultToolkit().getSystemClipboard().getContents(this);
-                return contents.isDataFlavorSupported(DataFlavor.stringFlavor);
+                Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+                if(null != clipboard) {
+                    Transferrable contents = clipboard.getContents(this);
+                    if(null != contents) {
+                        return contents.isDataFlavorSupported(DataFlavor.stringFlavor);

```

Comment:
Does Mirth Connect typically apply the only-one-return style or not?

Reply:
@jonbartels No, we don't typically apply the only-one-return coding style.